content/learn/book/getting-started/setup/_index.md

```diff
@@ -88,20 +88,20 @@ bevy = "0.5" # make sure this is the latest version
 Bevy can be built just fine using default configuration on stable Rust. However for maximally fast iterative compiles, we recommend the following configuration:
 
 * **Enable Bevy's Dynamic Linking Feature**: This is the biggest iterative compilation time win and requires no special setup. When you have `bevy` as a dependency you can run your app with dynamic linking like this:
-    ```bash
+    ```sh
```

Comment:
Changing from `bash` to `sh` (just shell) made no difference to any of the bevy-website text coloring, I just changed it to make consistent with the other changes.